pkg/odo/cli/pipelines/ui/ui.go

```diff
@@ -14,85 +15,79 @@ import (
 
 // EnterGitRepo allows the user to specify the git repository in a prompt
 func EnterGitRepo() string {
-	var gitopsUrl string
-	var prompt *survey.Input
-	prompt = &survey.Input{
+	var gitOpsURL string
+	prompt := &survey.Input{
 		Message: "Provide the URL for your GitOps repository",
 		Help:    "The GitOps repository stores your GitOps configuration files, including your Openshift Pipelines resources for driving automated deployments and builds.  Please enter a valid git repository e.g. https://github.com/example/myorg.git",
 	}
-
-	err := survey.AskOne(prompt, &gitopsUrl, survey.Required)
+	err := survey.AskOne(prompt, &gitOpsURL, survey.Required)
 	ui.HandleError(err)
 
-	return gitopsUrl
+	p, err := url.Parse(gitOpsURL)
```

Comment:
Are we validating the URL here? Shouldn't we do the same for the service repo URL?